tasks/debian_packages.yml

```diff
@@ -1,7 +1,24 @@
 ---
 - include_tasks: "{{ ansible_distribution | lower }}_repositories.yml"
 
+- name: Install linux headers (Ubuntu)
+  apt:
+    name:
+      - linux-headers-generic
```

Comment:
I'm not an ubuntu guy, are `headers-generic` required if you install kernel flavor specific headers (`linux-headers-{{ ansible_kernel }}`